omniduct/databases/hiveserver2.py

```diff
@@ -515,11 +518,11 @@ def _create_table_statement_from_df(cls, df, table, drop=False,
         {%- endif %}
         {%- if text %}
         ROW FORMAT DELIMITED
-        FIELDS TERMINATED BY "{{ sep }}"
```

Comment:
This change shouldn't break anything if done correctly, but currently the quotes are not correctly escaped any more. Since there's no difference, perhaps just revert these back to '"'?